src/Core/Common/GetImageSize/Documentation.rst

```diff
@@ -33,6 +33,13 @@ C++
 .. literalinclude:: Code.cxx
    :lines: 18-
 
+Python
+......
+
+.. literalinclude:: Code.py
+   :language: python
+   :lines: 1, 18-
```

Comment:
This could be a copy-paste mistake. @thewtex?

Reply:
Since `17-` is where the code starts, it should be `17-` instead of `18-`. The `1,` keeps the script she-bang.

Reply:
Done in f0247e0.